src/bootloader/bootloader.c

```diff
@@ -358,33 +358,44 @@ static void _render_progress(float progress)
 
 static void _render_hash(const char* title, const uint8_t* hash)
 {
-    uint8_t seconds = 10;
-    char message[16];
+    const uint8_t seconds = 10; // how many seconds to show screen
+    const UG_S16 title_margin = 7; // Margin between title and hash
+    const UG_FONT* f_mono = &font_monogram_5X9; // monospaced font
+    const UG_FONT* f_regular = &font_font_a_9X9; // regular font
+
+    char title_buf[50] = {0};
+    snprintf(title_buf, sizeof(title_buf), "%s hash", title);
+
+    // Convert hash to ascii hex
     char hash_hex[2 * SHA256_DIGEST_LENGTH + 1];
     util_uint8_to_hex(hash, SHA256_DIGEST_LENGTH, hash_hex);
-    char scratch = 0;
+
+    char timer_buf[3]; // Buffer for timer
```

Comment:
This will cut the number if it's > 10, right. Make a comment at `seconds` var that `10 is the max or increase timer_buf length`?

Reply:
Done.